_projects/2024/100535153/100535153.Rmd

```diff
@@ -0,0 +1,941 @@
+
+### Insert the data 
+
+I was unable to access an existing online dataset, so I had to recreate the data myself. Fortunately, a detailed description was provided on the same page as the article, and using that information, I was able to create an Excel file containing all the necessary details to produce the chart. To make it easier to manage the data across the corresponding graphs, I decided to organize the Excel file into separate sheets, each dedicated to a specific occupational area.
+
+Here you can find the link to the page of the article to extract the data: [The Changing Nature
+of Middle-Class Jobs - The New York Times ](https://www.nytimes.com/interactive/2015/02/23/business/economy/the-changing-nature-of-middle-class-jobs.html)
+
+```{r, echo = FALSE}
+data1 <- read_excel("C:/Users/marco/OneDrive/Desktop/Data Visualization - I sem/Project - The changing nature of middle-class job/Changing nature of middle class.xlsx")
+
+data2 <- read_excel("C:/Users/marco/OneDrive/Desktop/Data Visualization - I sem/Project - The changing nature of middle-class job/Changing nature of middle class.xlsx", sheet = "Managers and administrators")
+
+data3 <- read_excel("C:/Users/marco/OneDrive/Desktop/Data Visualization - I sem/Project - The changing nature of middle-class job/Changing nature of middle class.xlsx", sheet = "Service workers and salespeople")
+
+data4 <- read_excel("C:/Users/marco/OneDrive/Desktop/Data Visualization - I sem/Project - The changing nature of middle-class job/Changing nature of middle class.xlsx", sheet = "Technicians")
+
+data5 <- read_excel("C:/Users/marco/OneDrive/Desktop/Data Visualization - I sem/Project - The changing nature of middle-class job/Changing nature of middle class.xlsx", sheet = "Police, fire and military")
+
+data6 <- read_excel("C:/Users/marco/OneDrive/Desktop/Data Visualization - I sem/Project - The changing nature of middle-class job/Changing nature of middle class.xlsx", sheet = "Administrative support workers")
+
+data7 <- read_excel("C:/Users/marco/OneDrive/Desktop/Data Visualization - I sem/Project - The changing nature of middle-class job/Changing nature of middle class.xlsx", sheet = "Transportation workers")
+
+data8 <- read_excel("C:/Users/marco/OneDrive/Desktop/Data Visualization - I sem/Project - The changing nature of middle-class job/Changing nature of middle class.xlsx", sheet = "Construction and agricolture ")
+
+data9 <- read_excel("C:/Users/marco/OneDrive/Desktop/Data Visualization - I sem/Project - The changing nature of middle-class job/Changing nature of middle class.xlsx", sheet = "Skilled production workers")
+
+data10 <- read_excel("C:/Users/marco/OneDrive/Desktop/Data Visualization - I sem/Project - The changing nature of middle-class job/Changing nature of middle class.xlsx", sheet = "Machine operators and assembler")
```

Comment:
Put the datasets close to the Rmd and add them to the pull request. Then don't use paths in your computer, but just the names of the files.